editor/liveblocks.config.ts

```diff
@@ -82,6 +84,7 @@ export type ThreadMetadata = {
   type: 'canvas'
   x: number
   y: number
+  remixLocationPath?: string
```

Comment:
Minor, but can we call the location route? That is the terminology we mostly use.